services/web/server/tests/unit/with_dbs/conftest.py

```diff
@@ -78,20 +78,21 @@ def disable_swagger_doc_generation(
 
 
 @pytest.fixture(scope="session")
-def docker_compose_file(
-    default_app_cfg: ConfigDict, monkeypatch_session: pytest.MonkeyPatch
-) -> str:
-    """Overrides pytest-docker fixture"""
-
-    cfg = deepcopy(default_app_cfg["db"]["postgres"])
+def docker_compose_env(default_app_cfg: ConfigDict) -> Iterator[pytest.MonkeyPatch]:
+    postgres_cfg = default_app_cfg["db"]["postgres"]
 
     # docker-compose reads these environs
-    monkeypatch_session.setenv("TEST_POSTGRES_DB", cfg["database"])
-    monkeypatch_session.setenv("TEST_POSTGRES_USER", cfg["user"])
-    monkeypatch_session.setenv("TEST_POSTGRES_PASSWORD", cfg["password"])
+    with pytest.MonkeyPatch().context() as patcher:
+        patcher.setenv("TEST_POSTGRES_DB", postgres_cfg["database"])
```

Comment:
as discussed, this is equivalent to the current solution :-) but if  **for this case** you do not see any side-effect I guess we can live with it

Reply:
as I said, this is pytest-docker. there is no way around this.
Feel free to find another way.
My 2 cent here: get rid of pytest-docker. But I think we already had that discussion once.